security/nettle-tests.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import archive, build, distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Nettle(Test):
+
+    """
+    nettle-testsuite
+    :avocado: tags=security,testsuite,crypto
+    """
+
+    def setUp(self):
+        '''
+        Install the basic packages to support Nettle
+        '''
+
+        # Check for basic utilities
+        smm = SoftwareManager()
+        dd = distro.detect()
```

Comment:
having variable name length has minmum  size of 3 char is always good to have , let me enhance travis to catch this

Reply:
ok